### PR TITLE
Fonts substitution of PSA folder

### PR DESCRIPTION
**Pull request by BlackLotus64**
Change of fonts of PSA documents to MoneroSerif font set. Created some missing PDF files of the Portuguese translation from the SLA files.

### CODE REVIEW

File: monero-outreach-docs/translations/it/PSA/files/forks/forks_it.sla [hunk 1]
```diff
@@ -69,62 +69,59 @@
...
+            <ITEXT FONT="MoneroSerif Regular" CH="Questi progetti sono l'opposto di quelli menzionati nella sezione precedente. Sono motivati dall'avidità, sia da parte degli sviluppatori sia da parte degli utilizzatori (leggi: vittime). In alcuni casi, in particolare quelli in cui viene richiesto di condividere le proprie chiavi private, potrebbe trattarsi di un tentativo di attaccare la stessa rete Monero e di indebolire la privacy e l'anonimato dei suoi utilizzatori. "/>
             <para/>
             <para/>
-            <ITEXT FONT="Adobe Caslon Pro Italic" CH="Livello di sicurezza"/>
-            <ITEXT CH=": si sconsiglia caldamente di adottare queste monete. Procedi a tuo rischio e pericolo e sii cosciente che con il tuo comportamento potresti inavvertitamente indebolire la privacy e l'anonimato della rete Monero."/>
+            <ITEXT FONT="MoneroSerif Regular" CH="Livello di sicurezza: si sconsiglia caldamente di adottare queste monete. Procedi a tuo rischio e pericolo e sii cosciente che con il tuo comportamento potresti inavvertitamente indebolire la privacy e l'anonimato della rete Monero."/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/forks/forks_it.sla]: "Livello di sicurezza" is "MoneroSerif Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 2]
```diff
@@ -69,61 +69,61 @@
...
-            <ITEXT CH="[inglese]"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="monero.how"/>
+            <tab FONT="MoneroSerif Italic"/>
+            <tab FONT="MoneroSerif Italic"/>
+            <ITEXT FONT="MoneroSerif Italic" CH="[inglese]"/>
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="reddit.com/r/monero"/>
-            <tab/>
-            <ITEXT CH="[inglese]"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="reddit.com/r/monero"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 3]
```diff
@@ -69,61 +69,61 @@
...
+            <tab FONT="MoneroSerif Italic"/>
+            <ITEXT FONT="MoneroSerif Italic" CH="[inglese]"/>
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Guide to Monero (post)"/>
-            <tab/>
-            <ITEXT CH="[inglese]"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Guide to Monero (post)"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 4]
```diff
@@ -69,61 +69,61 @@
...
+            <tab FONT="MoneroSerif Italic"/>
+            <ITEXT FONT="MoneroSerif Italic" CH="[inglese]"/>
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero In-Depth Technical Intro"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero In-Depth Technical Intro"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 5]
```diff
@@ -69,61 +69,61 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Zero to Monero"/>
-            <tab/>
-            <tab/>
-            <ITEXT CH="[inglese]"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Zero to Monero"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 6]
```diff
@@ -69,61 +69,61 @@
...
+            <tab FONT="MoneroSerif Italic"/>
+            <tab FONT="MoneroSerif Italic"/>
+            <ITEXT FONT="MoneroSerif Italic" CH="[inglese]"/>
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Scams to Avoid"/>
-            <tab/>
-            <tab/>
-            <ITEXT CH="[inglese]"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Scams to Avoid"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 7]
```diff
@@ -69,61 +69,61 @@
...
+            <tab FONT="MoneroSerif Italic"/>
+            <tab FONT="MoneroSerif Italic"/>
+            <ITEXT FONT="MoneroSerif Italic" CH="[inglese]"/>
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero FAQ"/>
-            <tab/>
-            <tab/>
-            <ITEXT CH="[inglese]"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero FAQ"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 8]
```diff
@@ -69,61 +69,61 @@
...
+            <tab FONT="MoneroSerif Italic"/>
+            <tab FONT="MoneroSerif Italic"/>
+            <ITEXT FONT="MoneroSerif Italic" CH="[inglese]"/>
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Connect w/ Monero Community"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Connect w/ Monero Community"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 9]
```diff
@@ -69,61 +69,61 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Mastering Monero"/>
-            <tab FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="[inglese]"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Mastering Monero"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 10]
```diff
@@ -69,61 +69,61 @@
...
+            <tab FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla [hunk 11]
```diff
@@ -69,61 +69,61 @@
...
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="[inglese]"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/it/PSA/files/learning/learning_it.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/forks/forks_pt.sla [hunk 12]
```diff
@@ -69,30 +69,30 @@
...
+            <ITEXT FONT="MoneroSerif Regular" CH="Outros projetos usaram os &quot;hard fork&quot; regularmente programados do Monero em benefício próprio e criaram os seus próprios projetos nas antigas redes do Monero. Esses projetos normalmente falham em pouco tempo e  alimentam-se da ganância das pessoas. Se você não atualizar a sua carteira do Monero após um &quot;hard fork&quot; e enviar o seu XMR, poderá acabar perdendo-os na rede antiga ou ter que perder muito tempo para tentar recuperá-los."/>
             <para/>
             <para/>
-            <ITEXT FONT="Adobe Caslon Pro Italic" FONTSIZE="10.5" FSHADE="100" CH="Safety level:"/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH=" Best practice is to upgrade your Monero "/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="10.5" FSHADE="100" CH="Safety level:"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/forks/forks_pt.sla]: "MoneroSerif Italic"

File: monero-outreach-docs/translations/pt/PSA/files/forks/forks_pt.sla [hunk 13]
```diff
@@ -101,38 +101,38 @@
...
+            <ITEXT FONT="MoneroSerif Regular" CH="Estes projetos serão o oposto do que foi mencionado na seção de projetos legítimos. Estes projetos serão motivados pela ganância, tanto do lado dos programadores, como do lado dos utilizadores  (leia-se vítimas). Em alguns casos, particularmente no caso de projetos que solicitam aos utilizadores a partilha das chaves privadas, podemos estar perante uma tentativa de atacar a própria rede Monero e enfraquecer a privacidade / anonimato dos seus utilizadores."/>
             <para/>
             <para/>
-            <ITEXT FONT="Adobe Caslon Pro Italic" FONTSIZE="10.5" FSHADE="100" CH="Nível de segurança:"/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH=" Recomenda-se que se evite estas criptomoedas"/>
-            <ITEXT CH=". Proceda com muito risco e cuidado."/>
+            <ITEXT FONT="MoneroSerif Regular" FONTSIZE="10.5" FSHADE="100" CH="Nível de segurança:"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/forks/forks_pt.sla]: "MoneroSerif Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 14]
```diff
@@ -69,48 +69,49 @@
...
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="getmonero.org (Of"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" KERN="0" CH="f"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" KERN="-15" CH="i"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="cial Website)"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="getmonero.org (Of"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 15]
```diff
@@ -69,48 +69,49 @@
...
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" KERN="0" CH="f"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 16]
```diff
@@ -69,48 +69,49 @@
...
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" KERN="-15" CH="i"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 17]
```diff
@@ -69,48 +69,49 @@
...
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="cial Website)"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 18]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="monero.how"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="monero.how"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 19]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="reddit.com/r/monero"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="reddit.com/r/monero"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 20]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Guide to Monero (post) "/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Guide to Monero (post) "/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 21]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero In-Depth Technical Intro"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero In-Depth Technical Intro"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 22]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Zero to Monero"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Zero to Monero"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 23]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Scams to Avoid"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Scams to Avoid"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 24]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero FAQ"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Monero FAQ"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 25]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Connect w/ Monero Community"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Connect w/ Monero Community"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

File: monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla [hunk 26]
```diff
@@ -69,48 +69,49 @@
...
             <para LINESP="13.2"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Mastering Monero"/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="Mastering Monero"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt/PSA/files/learning/learning_pt.sla]: "MonereoSerif Bold Italic"

**Comment by BlackLotus64** [comment]
Resolved all occurrences and removed autosave files, thank you very much for your great review @lh1008.

File: monero-outreach-docs/en/PSA/files/learning/learning_en.sla [hunk 27]
```diff
@@ -68,56 +68,56 @@
...
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="getmonero.org (Of"/>
```
Comment (lh1008) [inline monero-outreach-docs/en/PSA/files/learning/learning_en.sla]: This font has not been fixed yet. [MoneroSerif-Bolditalic.otf](https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/fonts/monero%20fonts/MoneroSerif_v2/MoneroSerif-BoldItalic.otf)

File: monero-outreach-docs/translations/es/PSA/files/forks/forks_es.sla [hunk 28]
```diff
@@ -123,59 +126,59 @@
             <para LINESP="13"/>
             <para LINESP="13"/>
             <para LINESP="13"/>
-            <ITEXT FONT="Adobe Caslon Pro Italic" FONTSIZE="10" FSHADE="90" CH="Nivel de seguridad:"/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH=" "/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10" FSHADE="90" CH="Si es legítimo no necesitarás hacer nada con "/>
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="10" FSHADE="90" CH="Nivel de seguridad:"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/es/PSA/files/forks/forks_es.sla]: [MoneroSerif-Bolditalic.otf](https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/fonts/monero%20fonts/MoneroSerif_v2/MoneroSerif-BoldItalic.otf)

File: monero-outreach-docs/translations/fr/PSA/files/forks/forks_fr.sla [hunk 29]
```diff
@@ -21,13 +21,13 @@
...
         <PDF firstUse="0" Thumbnails="0" Articles="0" Bookmarks="0" Compress="1" CMethod="0" Quality="0" EmbedPDF="0" MirrorH="0" MirrorV="0" Clip="0" RotateDeg="0" PresentMode="0" RecalcPic="0" Grayscale="0" RGBMode="1" UseProfiles="0" UseProfiles2="0" Binding="0" PicRes="300" Resolution="300" Version="14" Intent="1" Intent2="0" SolidP="Apple RGB" ImageP="Apple RGB" PrintP="Artifex CMYK SWOP Profile" InfoString="" BTop="0" BLeft="0" BRight="0" BBottom="0" useDocBleeds="0" cropMarks="0" bleedMarks="0" registrationMarks="0" colorMarks="0" docInfoMarks="0" markOffset="0" ImagePr="0" PassOwner="" PassUser="" Permissions="-4" Encrypt="0" UseLayers="0" UseLpi="0" UseSpotColors="1" doMultiFile="0" displayBookmarks="0" displayFullscreen="0" displayLayers="0" displayThumbs="0" hideMenuBar="0" hideToolBar="0" fitWindow="0" PageLayout="0" openAction="">
-            <Subset Name="Adobe Caslon Pro Bold"/>
-            <Subset Name="Adobe Caslon Pro Italic"/>
-            <Subset Name="Adobe Caslon Pro Regular"/>
+            <Subset Name="MoneroSerif Bold"/>
```
Comment (lh1008) [inline monero-outreach-docs/translations/fr/PSA/files/forks/forks_fr.sla]: [MonereoSerif-Bolditalic.otf](https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/fonts/monero%20fonts/MoneroSerif_v2/MoneroSerif-BoldItalic.otf)

**Comment by lh1008** [comment]
This PR can be closed. We are actually changing the way to work the files so putting more energy to this could just be a waste of time. 

@BlackLotus64 and @erciccione.

**Comment by BlackLotus64** [comment]
Can this be finished and merged since there are only some fonts that need to be changed?

**Comment by lh1008** [comment]
Sure @BlackLotus64, you can finish it and we can merge it. No problem. :)

**Comment by lh1008** [comment]
I'm currently working on the .md files for the other languages too.

File: monero-outreach-docs/en/PSA/files/learning/learning_en.sla (hunk 27 above)
Comment (BlackLotus64) [reply monero-outreach-docs/en/PSA/files/learning/learning_en.sla]: Fixed. The forks files aren't supposed to contain the bold italic font if I'm not wrong.

**Comment by erciccione** [comment]
Is this ready for a final review @BlackLotus64 ?

**Comment by lh1008** [comment]
Thank you @erciccione

**Pull request merged**